src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -260,14 +275,41 @@ int halide_hexagon_remote_set_performance_mode(int mode) {
     mipsTotal = mipsPerThread * 2;
 
     bwMegabytesPerSec = bwBytePerSec >> 20;
-    return halide_hexagon_remote_set_performance(set_mips,
-                                                 mipsPerThread,
-                                                 mipsTotal,
-                                                 set_bus_bw,
-                                                 bwMegabytesPerSec,
-                                                 busbwUsagePercentage,
-                                                 set_latency,
-                                                 latency);
+
+    memset(&request, 0, sizeof(HAP_power_request_t));
+    request.type = HAP_power_set_apptype;
+    request.apptype = HAP_POWER_COMPUTE_CLIENT_CLASS;
+    retval = HAP_power_set(NULL, &request);
```

Comment:
It looks like this gets done twice now, if it falls through to set_performance. Is set_performance deprecated now? (I think it would be good if so, it's a complicated/hard to use API :) If set_performance is deprecated now, then I think it's OK this gets called twice, but if it is in fact deprecated, we should comment to that effect in HalideRuntimeHexagonHost.h.

Reply:
set_performance isn't deprecated. I will remove these calls as it these are already called for new DCVS_V2.